src/wilson/dom.cljs

```diff
@@ -13,12 +13,11 @@
              [:th (capitalize k)]))]
     (into [:tbody]
           (for [row rows]
-            (into [:tr
-                   {:class (row->cls row)}]
+            (into [:tr (or (row->attrs row) {})]
                   (for [k keys]
                     [:td (get row k)]))))])
   ([keys rows]
-   (table keys rows {:row->cls :wilson/row-class})))
+   (table keys rows {:row->attrs {}})))
```

Comment:
This doesn't make a lot of sense, although it does incidentally work. Here's what this means:

1. When you call this fn with two args, it calls itself for you with three args, and `{}` as `:row->attrs`
2. For each row, it will call `(row->attrs row)`, but `row->attrs` is `{}`, so that will be `({} row)`, which is of course always `nil`
3. `(or nil x) ;; => x`, so you get the default from the `or`.

You should consider replacing the `{}` with `(constantly {})`, and replace the `(or ...)` with just `(row->attrs row)`.